tests/functional/u/unsupported/unsupported_version_for_final.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+max_pyver=3.8
```

Comment:
This test is only run for Python < 3.8. We want to also test Python 3.8+ if `py-version=3.7`. I.e. the config should instead be
```conf
[master]
py-version=3.7
```